base_location/models/res_company.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    city_id = fields.Many2one(
+        'res.city',
+        compute='_compute_address',
+        inverse='_inverse_city_id',
+        string="City ID"
+    )
+    zip_id = fields.Many2one(
+        'res.city.zip',
+        string='ZIP Location',
+        compute='_compute_address',
+        inverse='_inverse_zip_id',
+        oldname="better_zip_id",
+        help='Use the city name or the zip code to search the location',
+    )
+    # In order to keep the same logic used in odoo, fields must be computed
+    # and inversed, not related. This way, we can ensure that it works
+    # correctly on changes and inconsistencies cannot happen.
+    # When you make the fields related, the constrains added in res.partner
+    # will fail. because when you change the city_id in the company, you are
```

Comment:
`s/./,`

Reply:
Can you elaborate on this?

Reply:
the dot is wrong, replace it w/ comma :)